packages/@lwc/integration-tests/scripts/wdio.sauce.conf.js

```diff
@@ -16,27 +16,36 @@ require('dotenv').config({
 
 const baseConfig = require('./wdio.conf.js');
 
+// #TODO[3754]: Only Chrome on Windows platform currently works, all other browser tests fail.
+// Need to investigate why sauce labs fails for all other platforms.
 const browsers = [
     // Note that headless Chrome also needs to be updated in wdio.conf.js for non-SauceLabs runs
     {
         commonName: 'chrome',
         browserName: 'chrome',
         version: 'latest',
+        // Note chrome fails for Linux and macOS
+        platform: 'Windows 11',
```

Comment:
Saucelabs recently swapped all our tests from running on Windows 10 to Linux which caused our tests to break.

I wasn't able to investigate further why this happens for Linux and macOS but pinning the platform to Windows 11 for now.

Created #3754 to investigate further.

Reply:
@nolanlawson could you review one more time for the ^ ?